.github/workflows/sync_branches.yml

```diff
@@ -0,0 +1,31 @@
+name: Sync Branches
+on:
+  push:
+    branches:
+      - "2.4"
+      - "2.5"
+  workflow_dispatch:
+
+jobs:
+  sync-branches:
+    strategy:
+      matrix:
+        include:
+          - from_branch: "2.4"
+            to_branch: "2.5"
+          - from_branch: "2.5"
```

Comment:
Would we always backport via the `n-1` version? What I mean by that is, if there is a change in `2.4`, would we expect `2.4 -> 2.5` PR to be reviewed and merged, then `2.5 -> main`? Or should we consider `2.4 -> 2.5` and `2.4 ->  main` to be created in parallel? 
I'm thinking, there could be case we the `2.4 -> 2.5` (or any stable versions) needs to be delayed due to issues or further work needed to support the backport, but in the meantime, there wouldn't be a need to hold it for the dev branch (`main`)

Reply:
I think we should almost always just go to the next branch, not open multiple PRs from the same source. This just leads to more noise and confusion.

If we ever need this, nothing is stopping us from opening a PR manually 😉